src/compose/ComposeMenu.js

```diff
@@ -100,7 +108,7 @@ class ComposeMenu extends PureComponent<Props> {
         waitUntilSaved: true,
       },
     };
-
+    requestCameraPermission();
```

Comment:
Is `READ_EXTERNAL_STORAGE` really necessary for just the camera? We shouldn't ask for it in this flow if it's not.

(Same with `CAMERA` in the `handleImagePicker` flow.)

Reply:
Even if we don't call ```READ_EXTERNAL_STORAGE```  app automatically asks for it on pressing the second time,it's unusual.